internal/integration_test/stdlibs/bench_test.go

```diff
@@ -91,20 +91,29 @@ var (
 			c = c.WithFSConfig(
 				wazero.NewFSConfig().
 					WithDirMount(sysroot, "/")).
-				WithEnv("PWD", normalizedTestdir)
+				WithEnv("PWD", normalizedTestdir).
+				WithEnv("GOWASIRUNTIME", "wazero")
 
 			args := []string{fname, "-test.short", "-test.v"}
 
+			// Some distributions of Go such as used by homebrew or GitHub actions do not
```

Comment:
We have a comment about homebrew in `Makefile` but it seems now setup-go on GHA also can run into the same problem on all OSes